### PR TITLE
Issue #868: Add credit to Savas Labs to footer

### PR DESCRIPTION
**Pull request by AnneTee**
@kostajh I added a credit to Savas in the footer. After some consideration I don't think adding our logo to the front page map makes sense, but we might add the footer text to the bottom of the "welcome" node that displays in the sidebar initially. :umbrella: :coffee: :chocolate_bar: :cloud:

### CODE REVIEW

File: themes/custom/mappy/templates/page--node.html.twig [hunk 1]
```diff
@@ -65,7 +65,13 @@
...
 <footer>
-  <div class="copyright">
-    <p>Copyright &copy; {{ 'now'|date('Y') }}</p>
+  <div class="footer__content">
+    <div class="footer__savas-credit">
+      <img src="/themes/custom/mappy/images/savas-logo.png" alt="Savas Labs logo">
+      <p>Website designed and built by <a href="http://savaslabs.com">Savas Labs</a></p>
+    </div>
+    <div class="footer__copyright">
+      <p>Copyright &copy; {{ 'now'|date('Y') }}</p>
```
Comment (kostajh) [inline themes/custom/mappy/templates/page--node.html.twig]: IANAL but should it say "Copyright 2016 Pauli Murray Project" ?

**Comment by kostajh** [comment]
Overall, looks good!

two thoughts:
- Would the Savas logo look OK if it was a little bit smaller? At first glance it seemed a bit out of proportion although I'm getting used to it
- yes to adding the footer text to the bottom of the welcome node in some way (maybe through a template specifically to that node ID)

**Comment by AnneTee** [comment]
@kostajh I'll update the copyright text and make the logo a bit smaller. I wondered if it was too eye-catching.

**Comment by AnneTee** [comment]
@kostajh I did those things. I'll figure out the node-specific addition next week!

**Comment by kostajh** [comment]
Merged! Leaving the Redmine issue open for now.

**Pull request merged**